### ISSUE TITLE
Could you allow custom mappings

### ISSUE THREAD

**Issue by CaptainQuirk** [pindexis/marker#11]
Hi,

__Ctrl + t__ is built-in in the terminal, so is __Ctrl + k__. Would it be possible for you to allow one to choose the keys the actions are bound to ?

Thanks in advance

**Comment by pindexis** [comment]
Yes you can. just set the desired key bindings in the following environment variables:
MARKER_KEY_MARK
MARKER_KEY_GET
MARKER_KEY_NEXT_PLACEHOLDER

here are the default ones:
```
# default key bindings
marker_key_mark="${MARKER_KEY_MARK:-\C-k}"
marker_key_get="${MARKER_KEY_GET:-\C-@}"
marker_key_next_placeholder="${MARKER_KEY_NEXT_PLACEHOLDER:-\C-t}"
```

**Comment by CaptainQuirk** [comment]
Hi,

Ok ! I'll be trying this ! Would you consider sourcing a ~/.markerrc file though ?
